micro-rdk/src/esp32/pulse_counter.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
what was the complain form clippy here?

Reply:
I think it was the unused code problem, but there was something I was doing weird that I fixed so this might not be an issue anymore. I'll check